README.md

```diff
@@ -30,24 +30,22 @@ The implementation is still in an early stage, but these are some of the (planne
       without worrying about collecting resources.
     - There are also challenge scenarios where you attempt to program
       robots in order to solve pre-designed puzzles or challenges.
-    - Future versions might also have multiplayer modes, with co-op or
-      PvP play over a network...?
 
-Installing and Playing
-======================
+Installing
+==========
 
-Warning: Swarm is still in an early stage; it's missing many of the
-planned features, there is no winning condition, and the gameplay is
-not very deep yet.  But you can program cute little robots, and hey,
-that's all that really matters, right?
+Instructions for installing from a binary release will go here!
```

Comment:
```suggestion
There will _soon_ be a release of Swarm on Hackage and in a binary form!

If you are interested in Haskell or functional programming, you can install
it from source (see below) or once it is released with the Cabal tool.

For instructions on how to install the binary directly check the releases.
```

Reply:
Oops, missed this comment too.  Will fix.